src/main/java/net/neoforged/neoforge/common/extensions/ILevelExtension.java

```diff
@@ -105,4 +108,17 @@ default void invalidateCapabilities(BlockPos pos) {}
      * but it is safe to call on any {@link Level}, without the need for an {@code instanceof} check.
      */
     default void invalidateCapabilities(ChunkPos pos) {}
+
+    /**
+     * Run a full single level tick.
+     *
+     * Wrapper method to expose the common method from {@linkplain ClientLevel} and {@linkplain ServerLevel},
+     * useful for ticking fake levels to update block/entity render animations.
+     *
+     * This method does not post the {@linkplain LevelTickEvent.Pre} or {@linkplain LevelTickEvent.Post} events,
+     * modders should post these events themselves before and after invoking this method.
+     */
+    default void tick(BooleanSupplier hasTime) {
```

Comment:
Wouldn't this break due to the server and the client tick method having different non-mapped names? Or does a recent neo change make this work now?

Reply:
mappings are not rly a thing any more since we run on and compile with official mojang mappings

Reply:
> Or does a recent neo change make this work now?
The removal of obfuscation as a thing (full mojmap at dev and runtime) means we can now hoist methods in this way.  We recently did this for some packet listener methods in #835 https://github.com/neoforged/NeoForge/blob/1.20.x/src/main/java/net/neoforged/neoforge/common/extensions/ICommonPacketListener.java#L28